model/eventGroupModel.js

```diff
@@ -11,10 +13,14 @@ const eventGroupSchema = Mongoose.Schema({
     },
     name: {
         type: String,
-        required: [true, 'Group name must be specified']
+        required: [true, 'Group name must be specified'],
+        minlength: [1, 'Event group name must be at least 1 character long'],
+        maxlength: [255, 'Event group name cannot be longer than 255 characters']
     },
     users: {
-        type: Array,
+        type: [{
+            type: SimpleUserSchema
+        }],
         required: [true, 'List of group users must be provided']
```

Comment:
I'm thinking maybe change this to `deafult`, this way you can create a group and add users later.